build-tests/recorder.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Record tests
+description: "A brand new test Recorder that collaborates with the AI Agent or works on its own"
+icon: "record-vinyl"
+---
+
+We have created a new Recorder to create test case in collaboration with the AI Agent or independently. Instead of just generating locator to copy into code (Playwright Codegen) it interacts with Octomind advanced features (e.g. variables) right from the recording window and finishes up your tests.
+
+## How to use Recorder
+
+1. Launch the Recorder (see options in the use cases below). A red-framed window opens.
+2. Use your mouse to select the element you want to interact with in the displayed snapshot.
+
+<Frame caption="Selecting the element we want to interact with in a Recorder snapshot, 05/2025">
+  <img
+    src="/images/build/recorder-1-select-element.png"
+    alt="Selecting the element we want to interact with in a Recorder snapshot"
+  />
+</Frame>
+
+3. Pick the desired interaction or assertion type.
+
+<Frame caption="Select the interaction or assertion type in Recorder, 05/2025">
+  <img
+    src="/images/build/recorder-2-select-type.png"
+    alt="Select the interaction or assertion type in Recorder"
+    width="400"
+  />
+</Frame>
+
+4. Expand to see advanced views
+
+In the lower right corner of the window you have access to options opening views with more details:
+
+- advanced setting displaying the locator code
+- recorded steps summary
+- their representation added to the prompt of the test case
+
+<Frame caption="Advanced view showing more details of the recording, 05/2025">
+  <img
+    src="/images/build/recorder-3-options.gif"
+    alt="Select the interaction or assertion type in Recorder"
+    width="400"
+  />
+</Frame>
+
+<div class="mt-8" />
+
+<Frame caption="Recorderd steps and their representation in the prompt views shown, 05/2025">
+  <img
+    src="/images/build/recorder-4-advanced-views.png"
+    alt="Recorderd steps and their representation in the prompt views shown"
+  />
+</Frame>
+
+The modals can be dragged by the six-dot icon and moved for better visibility.
+
+## Record test cases from anywhere
+
+Where can you use the Recorder?
+
+You can flexibly hand over between the Recorder and Agent during the **test generation process**, in the **editing** and the **repair mode** to help you re-record failed steps. You can record a **new test case from scratch**.
+
+### 1. Take over from the Agent during test generation
+
+When the Agent generates a test case it might take a path you dont want it to . You can stop it and take over from it . It open the Recorder at the spot the Agent left off and you can proceed recording next steps the test case.
```

Comment:
```suggestion
When the Agent generates a test case it might take a path you dont want it to. You can stop it and take over from it. It opens the Recorder at the spot the Agent left off and you can proceed recording  the next steps of the test case.
```